search/search.model.lkml

```diff
@@ -17,3 +17,49 @@ explore: +mobile_search_counts {
     SAP, and organic searches are those that occur directly on a search webpage (e.g. www.google.com).
     Warning: Firefox iOS is not able to implement all metrics, like ad clicks."
 }
+
+explore: +desktop_search_counts {
+  aggregate_table: rollup__search_clients_engines_sources_daily_submission_date__0 {
+    query: {
+      dimensions: [search_clients_engines_sources_daily.submission_date]
+      measures: [search_clients_engines_sources_daily.total_searches]
+      filters: [
+        search_clients_engines_sources_daily.source: "newtab",
```

Comment:
Without having much context on the data here, it looks like the other filters on `source` to have wildcards in them. Not sure if that should be the same here?
```suggestion
        search_clients_engines_sources_daily.source: "%newtab%",
```

Reply:
Good observation. I just checked and this rollup uses the filter with source specifically equal to `newtab` so this is intentional.